app/src/main/java/com/sschuraytz/charcoaldrawing/DrawingView.java

```diff
@@ -55,7 +55,10 @@ public boolean onTouchEvent(MotionEvent event) {
                 previousY = pointY;
                 break;
             case MotionEvent.ACTION_MOVE:
-                currentTool.drawContinuouslyBetweenPoints(undoRedo.getBitmapCanvas(), pointX, pointY, previousX, previousY);
+                // avoid continuously reprinting circle (thus making it darker) if user keeps finger on same spot
```

Comment:
Keeping the users finger on the same spot might be difficult in a real world environment whereas it's easy to do with a mouse on the emulator. I support this change however.